roles/engineblock/tasks/test.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create install directory
+  file: "path={{ engineblock_selenium_install_dir }}/selenium state=directory recurse=yes"
+
+- name: Install Selenium standalone
+  get_url:
+    url: "http://selenium-release.storage.googleapis.com/{{ engineblock_selenium_version | regex_replace('\\.[0-9]+$', '') }}/selenium-server-standalone-{{ engineblock_selenium_version }}.jar"
+    dest: "{{ engineblock_selenium_install_dir }}/selenium/selenium-server-standalone-{{ engineblock_selenium_version }}.jar"
+    mode: 0755
+
+- name: Install Chromedriver
+  unarchive:
+    src: "https://chromedriver.storage.googleapis.com/{{ engineblock_chromedriver_version }}/chromedriver_linux64.zip"
+    dest: "{{ engineblock_selenium_install_dir }}/selenium"
+    mode: 0755
+    copy: no
+
+- name: Install Chrome
```

Comment:
The installation of Chrome is now performed twice, as puppeteer also has this as a dependency and will also install it. In my opinion, installing it using OC-Deploy is the better solution. But it might be worthwile to check if puppeteer could be instructed to use this Chrome installation instead of pulling it in by itself.